avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -1248,4 +1250,52 @@ private void checkNonRecordConversionNull(Schema schema)
     Object converted = avroData.fromConnectData(schema, null);
     assertNull(converted);
   }
+
+  @Test
+  public void testToConnectDataStringEnumUnion() {
+    org.apache.avro.Schema enumSchema = org.apache.avro.SchemaBuilder.enumeration("TestEnum").symbols("A", "B", "C", "D");
+
+    Map<org.apache.avro.Schema, Object> avroSchemaToValueTests = new LinkedHashMap<>();
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.STRING), "AnotherValue");
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.INT), 1000);
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.LONG), 1000L);
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.BOOLEAN), false);
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.DOUBLE), Double.MAX_VALUE);
+    avroSchemaToValueTests.put(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.FLOAT), Float.MAX_VALUE);
+
+    for(Map.Entry<org.apache.avro.Schema, Object> kvp:avroSchemaToValueTests.entrySet()){
```

Comment:
@cotedm I typically use LinkedHashMap in tests because it makes debugging data issues much easier if you maintain order. In this case it's not necessarily required.